static/app/views/performance/database/getIntervalForMetricFunction.tsx

```diff
@@ -56,4 +56,4 @@ function registerGranularities(
 
 registerGranularities(COUNTER_AGGREGATES, COUNTER_GRANULARITIES);
 registerGranularities(DISTRIBUTION_AGGREGATES, DISTRIBUTION_GRANULARITIES);
-registerGranularities(SPAN_FUNCTIONS, COUNTER_GRANULARITIES);
+registerGranularities(METRICS_FUNCTIONS, COUNTER_GRANULARITIES);
```

Comment:
Does this change any functionality that others might be using? Because the `SPAN_FUNCTIONS` array contains a lot more elements than `METRICS_FUNCTIONS`. Maybe we should register both arrays?

Reply:
Ohh, this was an unintended change, let me change it back! 👍